pkg/core/localize/locales/en/cmd/context.en.toml

```diff
@@ -188,12 +188,14 @@ Use `rhoas context use-kafka` command to select a Kafka instance for your contex
 
 [context.common.error.registry.notFound]
 one='''
-Service registry instance doesn't exist, you need to set valid instance id using `rhoas context use-service-registry` command
+Service registry instance in your context doesnt exist.
+Your instance might have been removed outside rhoas-cli. 
```

Comment:
Not sure what it means.

Reply:
good point. Outside is bad word. Your instance have been removed sounds good overal

Reply:
```suggestion
Your instance might have been removed.
```